tests/report/host/host.cpp

```diff
@@ -108,17 +152,17 @@ int main(int argc, const char* argv[])
      * Host API tests.
      */
     g_enclave = enclave;
+
+#ifdef OE_USE_LIBSGX
     test_local_report(&target_info);
```

Comment:
Is it important to test local attestation here for USE_LIBSGX=OFF?

Reply:
Priority is to get it into master.  We can update this pretty easily once we have CI/CD running.

Reply:
Resolved.